flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestClientFactory.java

```diff
@@ -120,6 +127,40 @@ public void closeOpenChannelConnections(ConnectionID connectionId) {
 		}
 	}
 
+	void connectChannelWithRetry(ConnectingChannel connectingChannel, ConnectionID connectionId)
```

Comment:
nit: can be private if we do not plan to make it access out of this class

Reply:
Agree. Updated according.